bin/migrate

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env sh
```

Comment:
Do we want this to run in .circleci/bin/migrate?

Reply:
We could do. If so, I'd probably want to move `sync-opensearch-packages` too.